site/content/en/docs/api_sdk/sdk/developer-guide.md

```diff
@@ -12,21 +12,21 @@ the repository. To get the full package, one need to generate missing package fi
 
 ## Package file layout
 
-- `gen/` - generator files
 - `cvat_sdk/` - Python package root
 - `cvat_sdk/api_client` - autogenerated low-level package code
 - `cvat_sdk/core` - high-level package code
+- `cvat_sdk/gen/` - generator files
```

Comment:
There is no error here; these paths are just listed relative to the `cvat-sdk` directory. The only fix that is perhaps needed is to clarify that.

Reply:
> Ah, but folders below `gen` are relative to the root of repository.

No, they are not. Here's me listing directories from within the `cvat-sdk` directory:

```console
$ (cd cvat-sdk && find -mindepth 1 -maxdepth 2 -type d | sort)
./.openapi-generator
./cvat_sdk
./cvat_sdk/api_client
./cvat_sdk/auto_annotation
./cvat_sdk/core
./cvat_sdk/datasets
./cvat_sdk/pytorch
./docs
./docs/apis
./docs/models
./gen
./gen/templates
./requirements
```

As you can see, all of the directories mentioned in this documentation sections are here. So I think there just needs to be a small note explaining that this is the layout of the `cvat-sdk` directory.

> So, in the section of ## How to generate package code instead of current change there could be a note that the "below code is should be executed after moving into cvat-sdk/" although making this change and the current change made refer to the same thing.

I think in _that_ section it would actually be a good thing to amend the commands so that they can all be executed from the root of the repository. Currently, they seem to be inconsistent in that regard.

However, your current patch is incorrect: there is no `cvat_sdk` directory in the repository root. It should be `cvat-sdk`.

Reply:
Oh now I noticed that there's a difference of underscore and hyphen!